kde-settings-qubes.spec.in

```diff
@@ -125,5 +126,22 @@ if [ -r "$backupfile" ]; then
     mv -f "$backupfile" "$configfile"
 fi
 
+touch --no-create %{_kde4_iconsdir}/hicolor &> /dev/null ||:
+touch --no-create %{_kde4_iconsdir}/breeze &> /dev/null ||:
+
+%posttrans
+gtk-update-icon-cache %{_kde4_iconsdir}/hicolor &> /dev/null ||:
+gtk-update-icon-cache %{_kde4_iconsdir}/breeze &> /dev/null ||:
+update-desktop-database -q &> /dev/null ||:
+
+%postun
+if [ $1 -eq 0 ] ; then
+  touch --no-create %{_kde4_iconsdir}/hicolor &> /dev/null ||:
+  touch --no-create %{_kde4_iconsdir}/breeze &> /dev/null ||:
+  gtk-update-icon-cache %{_kde4_iconsdir}/hicolor &> /dev/null ||:
+  gtk-update-icon-cache %{_kde4_iconsdir}/breeze &> /dev/null ||:
+  update-desktop-database -q &> /dev/null ||:
+fi
```

Comment:
```suggestion
touch --no-create %{_kde4_iconsdir}/hicolor > /dev/null 2>&1 ||:
touch --no-create %{_kde4_iconsdir}/breeze > /dev/null 2>&1 ||:

%posttrans
gtk-update-icon-cache %{_kde4_iconsdir}/hicolor > /dev/null 2>&1 ||:
gtk-update-icon-cache %{_kde4_iconsdir}/breeze > /dev/null 2>&1 ||:
update-desktop-database -q > /dev/null 2>&1 ||:

%postun
if [ $1 -eq 0 ] ; then
  touch --no-create %{_kde4_iconsdir}/hicolor > /dev/null 2>&1||:
  touch --no-create %{_kde4_iconsdir}/breeze > /dev/null 2>&1 ||:
  gtk-update-icon-cache %{_kde4_iconsdir}/hicolor > /dev/null 2>&1||:
  gtk-update-icon-cache %{_kde4_iconsdir}/breeze > /dev/null 2>&1 ||:
  update-desktop-database -q > /dev/null 2>&1 ||:
fi
```

`&>` is a bashism

Reply:
Well, this is from the official Fedora packaging guidelines... But changing it makes sense anyway.